api/routes/test.ts

```diff
@@ -21,6 +21,9 @@ export async function testRoutes(fastify: FastifyInstance) {
           _res,
           done
         ) {
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore
```

Comment:
```suggestion
          // @ts-expect-error the types will be added
```
ts-expect-error is self-correcting. Once the error goes away, _it_ will create an error, forcing you to remove it.

Reply:
Thanks, I dint know about that comment.